### PR TITLE
Dropdown adjustments

### PR DESCRIPTION
**Pull request by philschanely**
## Description

Adds the following to dropdowns:

- Variation to expand to 100% of container
- Disabled styles for items

Screen Shot 2020-09-14 at 3 14 46 PM

### CODE REVIEW

File: lib/sage-frontend/javascript/system/dropdown.js [hunk 1]
```diff
@@ -56,6 +59,11 @@ Sage.dropdown = (function() {
       return;
     }
 
+    // Stop if the dropdown item clicked is disabled
+    if (el.classList.contains(dropdownDisabledItemClass)) {
+      return;
+    }
+
```
Comment (AndrwM) [inline lib/sage-frontend/javascript/system/dropdown.js:66]: Noice! Love the simplicity

File: lib/sage-frontend/stylesheets/system/patterns/objects/_dropdown.scss [hunk 2]
```diff
@@ -111,6 +117,12 @@ $-trigger-dropdown-offset: rem(16px) + sage-spacing(xs);
     padding-right: sage-spacing(md);
     margin-right: -1 * sage-spacing(2xs);
   }
+
+  .sage-dropdown__item--disabled &,
+  &:disabled {
+    color: sage-color(grey, 400);
+    pointer-events: none;
+  }
```
Comment (AndrwM) [inline lib/sage-frontend/stylesheets/system/patterns/objects/_dropdown.scss]: What do you think about adding a `cursor: not-allowed;` attribute to this?

**Review by AndrwM — APPROVED** [review]
Like and subscribe! +1

File: lib/sage-frontend/stylesheets/system/patterns/objects/_dropdown.scss (hunk 2 above)
Comment (philschanely) [reply lib/sage-frontend/stylesheets/system/patterns/objects/_dropdown.scss]: I'm fine with that. Sometimes I feel that cursor is a bit extra. But it certainly makes the possibilities (or lack thereof) obvious.

**Pull request merged**